wifiphisher/common/interfaces.py

```diff
@@ -160,6 +181,8 @@ def __init__(self, name, card_obj, mac_address):
         self._has_ap_mode = False
         self._has_monitor_mode = False
         self._is_wireless = False
+        self._is_managed_by_nm = False
```

Comment:
Oh yes I understand now. In that case I think it would be better to have another set `exclude_shutdown` where interfaces in there won't be touched on shutdown and then in the `is_interface_valid` if the interface is valid add the internet interface to this set as well. what do you think?